doc/source/user_guide/skimage2_migration.md

```diff
@@ -0,0 +1,30 @@
+(skimage2_migration)=
+
+# Transition to skimage2
+
+:::{hint}
+This document is a work in progress and still subject to change.
+:::
+
+scikit-image is preparing to release version 2.0 under the new namespace `skimage2`.
+This will affect both its import name and its package name on PyPI and elsewhere.
+Together with skimage2, we will release version 1.0.0 and 1.0.1 with the old API.
+Version 1.0.1 will emit a notification (FutureWarning) on import, telling users to either upgrade to skimage2 or pin to version 1.0.0.
```

Comment:
I realize now we can't do this, because then we cannot make patch releases to close bugs. So, we'd need to figure out a different scheme.

Maybe 0.XX.Y is the last version without warnings, and 1.0.0 the one with warnings. Or 1.0.X the last without warnings and 1.1.X the one with warnings.

Reply:
That is a very good point. Maybe we can hack something together using [pre-releases](https://packaging.python.org/en/latest/specifications/version-specifiers/#handling-of-pre-releases) since they get special treatment from pip?